src/Umbraco.Cms.Integrations.Automation.Zapier/Controllers/PollingController.cs

```diff
@@ -97,43 +96,25 @@ public IEnumerable<PublishedContentDto> GetSampleContent()
             });
         }
 
-        public List<Dictionary<string, string>> GetContentByAlias(string contentTypeAlias)
+        public List<Dictionary<string, string>> GetContentByType(string alias)
         {
             var list = new List<Dictionary<string, string>>();
 
             using (var cref = _umbracoContextFactory.EnsureUmbracoContext())
             {
                 var cache = cref.UmbracoContext.Content;
                 
-                var nodes = cache.GetByXPath($"//{contentTypeAlias}")
+                var node = cache.GetByXPath($"//{alias}")
```

Comment:
This doesn't look quite right to me still.  Would we not be better here using the `IContentTypeService` to retrieve the content type (not the content of a provided type, which may not exist)?

When we have that we can format a response (using the current date for the published time, and empty strings for the property values).

Reply:
Updated.